Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -962,6 +962,22 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Verify if string matches the regex pattern and return corresponding parts of string.
+    ///
+    /// - Parameter pattern: Pattern to verify.
+    /// - Returns: Matching pattern parts of string.
+    /// - Throws: Throws any errors thrown by creation of invalid pattern.
+    func matchings(for pattern: String) throws -> [String] {
```

Comment:
```suggestion
    func matches(for pattern: String) throws -> [Substring] {
```
"Matchings" is not a word in English, so I think you should call this `matches`, as it's valid and won't conflict with the previous definition since you change the parameter name.

I also think that the return value should be the substrings as it's more performant.